filestore/handlers_base.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import six
+import logging
+
+
+class HandlerBase(object):
+    """
+    Base-class for Handlers to provide the boiler plate to
+    make them usable in context managers by provding stubs of
+    ``__enter__``, ``__exit__`` and ``close``
+    """
+    specs = set()
```

Comment:
Yeah, I recognized the pims code. I'm wondering whether it would be safer to make this a property. Or maybe we'll shrug and say, "Python is a language for contenting adults." Mutate at your own risk. That's fine.